ts/components/messages/MessageDetailComponent.tsx

```diff
@@ -103,15 +112,12 @@ export default class MessageDetailComponent extends React.PureComponent<Props> {
           containerStyle={styles.ctaBarContainer}
         />
 
-        <View style={styles.markdownContainer}>
-          <Markdown
-            lazyOptions={{ lazy: true }}
-            initialState={{ screen: "MESSAGE_DETAIL" }}
-          >
-            {message.content.markdown}
-          </Markdown>
-        </View>
-      </View>
+        <MarkdownViewer
+          markdown={message.content.markdown}
+          htmlBodyClasses={"message--detail"}
```

Comment:
could you document the usage of this parameter? I don't see `message--detail` defined anywhere